webclient/app/src/app/commons/templateCard/TemplateCard.jsx

```diff
@@ -65,24 +62,24 @@ function TemplateCard(props) {
                 <Grid container spacing={0}>
                     <Grid item xs={7}>
                         <Typography gutterBottom variant="h5" component="span">
-                            {template.name}
+                            {appTemplate.name}
                         </Typography>
                     </Grid>
                     <Grid item xs={5} align="right">
-                        <IconButton><Edit /></IconButton>
-                        <IconButton><Delete /></IconButton>
+                        <IconButton><Edit onClick={() => handleEdit(appTemplate)} /></IconButton>
```

Comment:
I think the onClick handler should be placed on the IconButton tag before the edit icon.